pkg/ttl/ttlworker/session_test.go

```
@@ -162,16 +164,137 @@ func newMockSessionPool(t *testing.T, tbl ...*cache.PhysicalTable) *mockSessionP
+	default:
+		loc, err := time.LoadLocation(timeZone)
+		require.NoError(s.t, err)
+		s.sessionVars.TimeZone = loc
+	}
+}
+
+func (s *prepareSessionMock) setIsolationReadEngines(value string) {
+	s.isolationReadEngines = value
+	s.sessionVars.IsolationReadEngines = make(map[kv.StoreType]struct{})
+	for _, engine := range strings.Split(value, ",") {
+		switch strings.TrimSpace(engine) {
```

golangci-lint (2.12.2)

[error] enforce-switch-style: switch must have a default case clause

(revive)